newsroom/reports/reports.py

```diff
@@ -87,28 +87,24 @@ def get_user_saved_searches():
 def get_company_products():
     """Returns products by company"""
     results = []
-    company_products = {}
     companies = get_entity_dict(query_resource("companies"))
-    products = query_resource("products")
+    products_data = list(query_resource("products"))
+    for company_id, details in companies.items():
```

Comment:
`details` here doesn't describe what entity this is talking about. Maybe renaming it to something like `company` would be more meaningful